tests/testthat/test_BART_surv_bart.R

```diff
@@ -0,0 +1,8 @@
+test_that("autotest", {
+  learner = lrn("surv.bart", nskip = 1, ndpost = 1, keepevery = 1)
+  expect_learner(learner)
+  # `feat_all` task has a factor which is split to two variables which give two
```

Comment:
I think this is problematic. The `$importance()` method should always return one score for each feature.

Reply:
at least it should give a good error message when a factor variable was present

Reply:
decided to remove the factor type variables. Users can encode them easily before.